support/nginx/peertube

```diff
@@ -57,6 +57,26 @@ server {
     root /var/www/certbot;
   }
 
+  # Bypass PeerTube for performance reasons. Could be removed
+  # Must be consistent with static client-overrides list in /server/controllers/client.ts
+  location ~ ^/client/(assets/images/(logo\.svg|favicon\.png)) {
```

Comment:
```suggestion
  location ~ ^/client/assets/images/(logo\.svg|favicon\.png)$ {
```

Reply:
@rigelk are your sure your suggestion would also catch `logo.svg?569beefcd2eea8ed70010c70310317f33b5fade2` ?

Reply:
Ah I see, my bad, I forgot the regex group was used

Reply:
btw it's working event with the `$`, nginx seems to not see query strings at this point.